catalog/ui/search-ui/simple/pom.xml

```diff
@@ -133,7 +133,8 @@
                             javax.servlet,
                             org.osgi.framework,
                             org.slf4j,
-                            javax.servlet.http
+                            javax.servlet.http,
```

Comment:
❗️ This does not fix issue. Mirror this: https://github.com/mweser/ddf/commit/a25233cb8530fcb00ad2d59a7111c1b36e9dc75c